packages/terra-date-input/src/terra-dev-site/test/date-input/common/_RadioFieldGenerator.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Radio from 'terra-form-radio';
+import RadioField from 'terra-form-radio/lib/RadioField';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  // The text of the legend tag.
+  fieldLegend: PropTypes.string,
+  // The name of every Radio
```

Comment:
Period for consistency
```suggestion
  // The name of every Radio.
```